.gitpod.yml

```diff
@@ -0,0 +1,15 @@
+---
+image:
+  file: .gitpod/Dockerfile
+  context: .gitpod
+
+tasks:
+  - init: make
```

Comment:
As is, this isn't really going to do much (since `help` is the current default target)... `make build-local-infrastructure` will likely give us bigger bang for the buck.